frontend/src/routes/Governance/policies/policy-details/PolicyTemplateDetails.tsx

```diff
@@ -298,7 +306,7 @@ export function PolicyTemplateDetails() {
         sort: 'apigroup',
         search: 'apigroup',
       },
-      violationColumn,
+      ...(isVAPB ? [] : [violationColumn]),
```

Comment:
Which column are u going to use? If relatedResourceColumns you need to modify line 403 and line 43 or If relatedResourceFromSearchAPIColumns you need to remove this line

Reply:
recommend use relatedResourceColumns

Reply:
for rebase

Reply:
I wanted to remove the violation column for parameter resources. I followed what I think Matt did for Kyverno.

I don't follow how I would use `relatedResourceColumns`, since I'm trying to display things from the search api...

Reply:
Also then we should remove   line 243  `  ...(kind === 'ValidatingAdmissionPolicyBinding' ? [] : [violationColumn]),`

Reply:
For most policy types (including eg Kyverno ClusterPolicy), we want to show the violations column. But specifically for ValidatingAdmissionPolicyBinding, the table will be showing the parameter resources, which don't have a violation status, right? So the idea was to remove that column for those pages.

Reply:
So for vapb with columns are you gonna use relatedResourceFromSearchAPIColumns? or relatedResourceColumns

Reply:
relatedResourceFromSearchAPIColumns